.github/workflows/style.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install tox and pre-commit
+        run: pip install tox pre-commit
```

Comment:
Should not have to install `pre-commit`, tox will do that for you.

Reply:
Yeah, sorry, it is an artifact from trying to make this workflow to pass